### DISCUSSION TITLE
The react-native document does not even have a monitoring document for whether it has been connected or disconnected, and it was not found in the demo.

### DISCUSSION THREAD

**Discussion by a943999733** [WalletConnect/walletconnect-monorepo#2994] [Developer Support] [answered]
The react-native document does not even have a monitoring document for whether it has been connected or disconnected, and it was not found in the demo.
The dapp actively disconnects the wallet and cannot obtain the monitoring

**Comment by ignaciosantise** [comment, marked as answer]
hey @a943999733 👋 can you explain more what's your issue? And provide reproducible code.

If you want to track the connection between a wallet and your dapp, you can use `isConnected` prop.

`const { isConnected } = useWalletConnectModal()`

Remember that if you reopen the app, it takes some seconds to restore the session again.

You can read more here: https://docs.walletconnect.com/2.0/reactnative/walletconnectmodal/dapp-usage#usewalletconnectmodal
